### ISSUE TITLE
histogram OOMs after only 2M entries or so

### ISSUE THREAD

**Issue by eigenhombre** [incanter/incanter#211]
```
(let [data (repeat 2000000 (rand-int 10))]
  (histogram data :nbins 10))
```

Gives an out-of-memory error;

```
(let [data (repeat 1000000 (rand-int 10))]
  (histogram data :nbins 10))
```

does not.  Maybe this is in JFreeChart (hard to fix); but I need to make some large-ish histograms and this is a show-stopper for that at the moment.

Thanks!

**Comment by MawiraIke** [comment]
> After further investigation, the (or at least one) place where the OOM occurs seems to be in the `(double-array _x)` part of `histogram*`.

True, i think so too. I am not sure if this is a fix but did you try adding

``` clojure
:jvm-opts ["-Xmx1G"]
```
in your project.clj.
Again I am not sure if this is a fix but i read somewhere that ```the secret to running out of memory boils down to giving the JVM more memory```.

Adding the above will make Leiningen launch the JVM with a higher memory cap and maybe that could be the issue.
